rosidl_generator_py/cmake/custom_command.cmake

```diff
@@ -23,7 +23,8 @@ add_custom_command(
   COMMAND ${PYTHON_EXECUTABLE} ${rosidl_generator_py_BIN}
   --generator-arguments-file "${generator_arguments_file}"
   --typesupport-impls "${_typesupport_impls}"
-  DEPENDS ${target_dependencies} ${rosidl_generate_interfaces_TARGET}
+  ${extra_generator_arguments}
```

Comment:
Where is this variable being defined?

Reply:
A left over from a previous attempt. Thanks for pointing it out. Removed.